### ISSUE TITLE
Train từ pretrain mới

### ISSUE THREAD

**Issue by lynguyenminhuit** [pbcquoc/vietocr#57]
Chào anh.

Tập dataset của e ~70k ảnh. Em dùng model của a train khoảng 40k iters.
Do train trên colab, khoảng 4h là nó disconnect. E muốn train tiếp từ pretrain vừa rồi. E có sửa: 
config['trainer']['checkpoint'] = 'path tới file .pth vừa save'
config['pretrain']['cached'] = 'path tới file .pth vừa save'
E không biết sửa ở đó đúng chưa ạ, mà khi train thì loss nó tương đương với khi train bằng pretrain gốc của a.

**Comment by pbcquoc** [comment]
hi e,
e tự load thủ công nha. 
```
trainer.model.load_state_dict(torch.load('model.pt'))
```
cách này sẽ load lại trọng số và train lại từ đầu dựa trên trọng số đó.

**Comment by lynguyenminhuit** [comment]
> hi e, e tự load thủ công nha.
> 
> ```
> trainer.model.load_state_dict(torch.load('model.pt'))
> ```
> 
> cách này sẽ load lại trọng số và train lại từ đầu dựa trên trọng số đó.

E train dc r a a ơi
E cảm ơn a nhiều ạ.
